sycl/doc/CompilerAndRuntimeDesign.md

```diff
@@ -799,6 +799,7 @@ For instance:
 // GPU/accelerator implementation
 template <typename T, address_space AS> class multi_ptr {
   // GetAnnotatedPointer<T, global>::type == "__attribute__((opencl_global)) T"
```

Comment:
Why is __attribute__ using bold font?

Reply:
I don't know all the nuts and bolts of GitHub rendering engine for Markdown documents, but I guess double underscores have something to do with this.

Reply:
We'd rather fix this because otherwise it looks like calling it out for something.

Reply:
Do you have particular suggestion how to do this?

Reply:
BTW, the bold font is visible only in diff view. There is no bold in rendered view: 
- https://github.com/intel/llvm/blob/af73e40f62c9805e99163ae2a6833b92c7b740ff/sycl/doc/CompilerAndRuntimeDesign.md#address-spaces-handling
- https://intel.github.io/llvm-docs/CompilerAndRuntimeDesign.html#lowering-of-lambda-function-objects-and-named-function-objects